stratisd_cert.py

```diff
@@ -90,14 +90,19 @@ def make_test_filesystem(pool_path, fs_name):
     return array_of_tuples_with_obj_paths_and_names[0][0]
 
 
-def acquire_filesystem_symlink_targets(pool_path, filesystem_path):
+# pylint: disable=bad-continuation
+def acquire_filesystem_symlink_targets(
+    pool_name, filesystem_name, pool_path, filesystem_path
```

Comment:
Whoops! I missed a spot. You should be able to put the ```disable``` after the open paren, like here https://github.com/stratis-storage/stratisd/blob/develop-2.3.0/tests/client-dbus/tests/udev/test_udev.py#L265.

Reply:
I tried that before, and it didn't work:
```
************* Module stratisd_cert
stratisd_cert.py:94:0: C0330: Wrong hanging indentation before block (add 4 spaces).
    pool_name, filesystem_name, pool_path, filesystem_path
    ^   | (bad-continuation)
```

Reply:
Now it seems to have worked; I updated it via a force-push.